lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -276,39 +314,47 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
     }
 
     func textFieldShouldReturn(_ textField: UITextField) -> Bool {
-        if let nextTextField = textField.superview?.superview?.viewWithTag(textField.tag + 1) as? UITextField {
+        guard let textFields = addComicViewControllerTextFields else { return false }
+        guard let currentTextFieldArrayIndex = textFields.index(of: textField) else { return false }
```

Comment:
This assignment can be grouped with the previous `guard` since it's doing the same thing on return